rust/agama-autoinstall/src/scripts.rs

```diff
@@ -60,6 +62,9 @@ impl ScriptsRunner {
     /// It downloads the script from the given URL to the runner directory.
     /// It saves the stdout, stderr and exit code to separate files.
     ///
+    /// If the script cannot be execute due to the error "Text file busy (os error 26)",
+    /// wait for 50 milliseconds and try again (up to 5 times).
```

Comment:
well, as exact sleep waiting is internal details of run_with_retry and also retry count. I would probably not specify here and just mention that it use run_with_retry where it will be documented.